dataplane/src/packet_processor/ingress.rs

```diff
@@ -19,7 +19,7 @@ use routing::interfaces::interface::{Attachment, IfState, IfType, Interface};
 use tracectl::trace_target;
 trace_target!("ingress", LevelFilter::WARN, &["pipeline"]);
 
-#[allow(unused)]
+#[derive(Debug)]
```

Comment:
Why do we want to derive Debug?

Reply:
I'm not sure if we use it now, but I don't think it does any harm if we have it and use it in the future right?

Reply:
It does not harm I guess (other than increasing build times?) :shrug: . I was just curious if there was a specific reason.